### PR TITLE
Setup container:pull command

### PR DESCRIPTION
**Pull request by dmathieu**
Usage:

```
heroku container:pull web
```

```
heroku container:pull web worker
```

```
heroku container:pull web:tag
```

### CODE REVIEW

**Comment by dmathieu** [comment]
cc @jbyrum

File: commands/pull.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const cli = require('heroku-cli-util')
+const Sanbashi = require('../lib/sanbashi')
+const debug = require('../lib/debug')
+
+let usage = `
+    ${cli.color.bold.underline.magenta('Usage:')}
+    ${cli.color.cmd('heroku container:pull web')}        # Pulls the web image from the app
+    ${cli.color.cmd('heroku container:pull web worker')} # Pulls both the web and worker images from the app
+    ${cli.color.cmd('heroku container:push web:latest')} # Pulls the latest tag from the web image`
```
Comment (tt) [inline commands/pull.js:9]: Do we want to advertise how to use tags? It's currently unspecified how they work with the registry.

**Review by tt — APPROVED** [review]

File: commands/pull.js (hunk 1 above)
Comment (dmathieu) [reply commands/pull.js:9]: I've set it based on the current RFC we have. But we can add them later based on how we end up doing the rest of the work.
@jbyrum what do you think?

Comment (jbyrum) [reply commands/pull.js:9]: How would customers know what tagged images are in the registry?

Comment (dmathieu) [reply commands/pull.js:9]: Looks like the only way is to make an HTTP call to the registry: https://stackoverflow.com/a/31750543/122080
We could make that a `container:list` command.

Comment (jbyrum) [reply commands/pull.js:9]: Just closing the loop on this -- we discussed in our team meeting to not support pulling tags at this time

Comment (jbyrum) [reply commands/pull.js:9]: Also, the third help command says `push` when it should say `heroku container:pull`

**Pull request merged**